conf.py

```diff
@@ -264,7 +264,7 @@
 # Nikola supports logo display.  If you have one, you can put the URL here.
 # Final output is <img src="LOGO_URL" id="logo" alt="BLOG_TITLE">.
 # The URL may be relative to the site root.
-LOGO_URL = '/images/quansight_labs_logo.png'
+LOGO_URL = '/images/quansight_labs_logo_white.png'
```

Comment:
I think the file of the white logo image is missing in the commits, that's why it's doing a fallback to the `alt` value `Quansight Labs` instead of showing the image